docker_images/sklearn/app/pipelines/common.py

```diff
@@ -34,7 +34,10 @@ def __init__(self, model_id: str):
                 # this is the default path for configuration of a scikit-learn
                 # project. If the project is created using `skops`, it should have
                 # this file.
-                config = json.load(f)
+                try:
+                    config = json.load(f)
+                except Exception:
+                    raise ValueError("`config.json` is invalid.")
```

Comment:
I don't think this will fail as you expect as it will be catched by the outer try..except.

What you want I think it removing the outer try..except (which ignored `config.json` when it's wrong instead of failing).